src/composables/useAreaOfInterest.ts

```diff
@@ -188,18 +188,7 @@ export default function useAreaOfInterest() {
     extentInteraction.value = null
   }
 
-  function clearResultsAndZoomToGrid(map: Map) {
-    geoJsonResults.value = []
-
-    // Remove the GeoJSON results layer from the map
-    if (vectorLayer.value) {
-      map.removeLayer(vectorLayer.value)
-      // Dispose of the layer source to free memory
-      if (vectorLayer.value.getSource) {
-        vectorLayer.value.getSource()!.dispose()
-      }
-    }
-
+  function returnToResults(map: Map) {
```

Comment:
Why did we set maxZoom to 13 below? It seems a bit too far out for smaller results. In other places we have higher maxZooms set.